test/smoke/host_teams_numa_affinity/Makefile

```diff
@@ -0,0 +1,26 @@
+include ../../Makefile.defs
+
+TESTNAME     = host_teams_numa_affinity
+TESTSRC_MAIN = host_teams_numa_affinity.c
+TESTSRC_AUX  =
+TESTSRC_ALL  = $(TESTSRC_MAIN) $(TESTSRC_AUX)
+
+TARGET       =
```

Comment:
doesnt this need 
 TARGET = -fopenmp -O2

Reply:
I think we do not need TARGET = -fopenmp -O2

I ran make run and I am getting -O2 -fopenmp already.

aomp/test/smoke/host_teams_numa_affinity$ make run
Makefile:23: warning: overriding recipe for target 'run'
../Makefile.rules:30: warning: ignoring old recipe for target 'run'
~/rocm/aomp/bin/clang  -O2    -fopenmp   host_teams_numa_affinity.c -o host_teams_numa_affinity